d3ledger/back-office
# D3 1044 | Reports

# Description
https://soramitsu.atlassian.net/browse/D3-1044

In order to correct work, has changes from https://github.com/d3ledger/back-office/pull/181

# Works done
- [x] Custody billing report
- [x] Transfer billing report
- [x] Exchange billing report

# How to test 
0. run back-office-admin, back-office, notary
1. Set fee in admin
2. Make all basic actions 
3. Check report page
